sys/net/application_layer/emcute/include/emcute_internal.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_emcute
+ * @{
+ *
+ * @file
+ * @brief       emCute internals
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef EMCUTE_INTERNAL_H
+#define EMCUTE_INTERNAL_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   MQTT-SN message types
+ */
+enum {
+    ADVERTISE       = 0x00,     /**< advertise message */
+    SEARCHGW        = 0x01,     /**< search gateway message */
```

Comment:
nope it was the other way around, that flags shouldn't be defined as enums, but defines. But these are no flags, but field values.